.flake8

```diff
@@ -1,5 +1,6 @@
 [flake8]
 # https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html
 # Ignore "Module level import not at top of file (E402)"
-ignore=E402
+# Use `docker run -ti --rm -v $(pwd):/apps alpine/flake8 -v plugins/modules/*` to validate
+ignore=E402,W504,W503
```

Comment:
Why is this needed ?

Reply:
I don't like warning, and this warning is in conflict.
https://www.flake8rules.com/rules/W503.html
https://www.flake8rules.com/rules/W504.html
maybe we can keep check on w504 and ignore W503

Reply:
I don't like warnings neither, but… I don't see any warnings :D

Reply:
I have this warning : 

```
plugins/modules/dedicated_nasha_manage_partition.py:288:17: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:334:30: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:339:17: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:347:13: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:386:25: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:462:13: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:467:9: W503 line break before binary operator
plugins/modules/dedicated_nasha_manage_partition.py:468:9: W503 line break before binary operator
```

Reply:
Meh, I don't see anything :thinking: 
```bash
$ docker run -ti --rm -v $(pwd):/apps alpine/flake8 --version
7.0.0 (flake8-sarif: 1.0.15, mccabe: 0.7.0, pycodestyle: 2.11.1, pyflakes: 3.2.0) CPython 3.12.3 on Linux
$ cat .flake8 
[flake8]
# https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html
# Ignore "Module level import not at top of file (E402)"
ignore=E402
max-line-length = 150
 $ docker run -ti --rm -v $(pwd):/apps alpine/flake8 -v plugins/modules/*
flake8.checker            MainProcess    169 INFO     Making checkers
flake8.main.application   MainProcess    238 INFO     Finished running
flake8.main.application   MainProcess    238 INFO     Reporting errors
flake8.main.application   MainProcess    238 INFO     Found a total of 33 violations and reported 0
```

The 33 violations are from E402 (ignored)

Reply:
on which branch ?